lib/resty/etcd/v3.lua

```diff
@@ -160,6 +190,37 @@ local function choose_endpoint(self)
     return endpoints[pos]
 end
 
+-- return refresh_is_ok, error
+refresh_jwt_token = function (self)
```

Comment:
`refresh_jwt_token = function (self)` 
change to:
 `function refresh_jwt_token(self)`

Reply:
fixed